paper_trail.gemspec

```diff
@@ -19,9 +19,7 @@ has been destroyed.
   s.email = "jared@jaredbeck.com"
   s.license = "MIT"
 
-  s.files = `git ls-files -z`.split("\x0").select { |f|
-    f.match(%r{^(Gemfile|LICENSE|lib/|paper_trail.gemspec)})
-  }
+  s.files = Dir.glob(["lib/**/*", "LICENSE"])
```

Comment:
Thank you for this change!  Can we please include `Gemfile` and `paper_trail.gemspec`?

Reply:
Judging by the title, this looks like it was on purpose.  I understand removing the git dependency but why remove `Gemfile` and the gemspec?

Reply:
Well, honestly, shipping Gemfile and gemspec doesn't make much sense. There's, of course, no harm is including them but it is redundant, really! 😄

Reply:
Oh okay, it sounds like I'm missing something.  Don't we need the Gemfile to load the gemspec?

Reply:
Yes, but only to build the package. What you mention in the `s.files` field are the things that will be shipped to the end user. For an end user, having Gemfile and gemspec is rather redundant.

Reply:
OTOH, docs and README are something that could benefit the end users, so I'd probably ship that.

But since it wasn't shipped earlier, I'll just leave it as a slight suggestion 😄

Reply:
I see, thanks for explaining!

Hi @jaredbeck, what do you think of:
- Updating this to not depend on `git`
- Remove `Gemfile` and the gemspec
- Include the Readme and Changelog